### PR TITLE
Added some clarity to the initial setup section

### PR DESCRIPTION
**Pull request by mcough2**
### Proposed changes

Added some clarity to the initial setup section to follow Metronome docs
https://docs.metronome.com/getting-usage-data-into-metronome/using-segment-to-ingest-events/

### Merge timing
ASAP once approved?

### CODE REVIEW

File: src/connections/destinations/catalog/actions-metronome/index.md [hunk 1]
```diff
@@ -11,13 +11,13 @@ redirect_from:
 
 Metronome also enables product-led growth with a consistent source of truth for use and billing. Freely experiment with pricing and packaging and put iteration directly in the hands of your Product team.
 
-## Getting Started
+## Initial Setup
```
Comment (markzegarelli) [inline src/connections/destinations/catalog/actions-metronome/index.md:14]:
```suggestion
## Getting Started
```

For consistency with the rest of our integrations docs, we'll leave this as `Getting Started` for now. We're looking at a broader update here though to make things more clear.

File: src/connections/destinations/catalog/actions-metronome/index.md [hunk 2]
```diff
@@ -11,13 +11,13 @@ redirect_from:
...
 
 1. From the Segment web app, click **Catalog**, then click **Destinations**
-2. Search for **Metronome (Actions)** within the Destinations Catalog and select **Metronome (Actions)**
+2. Search for ''Metronome (Actions)'' within the Destinations Catalog and select **Metronome (Actions)**
```
Comment (markzegarelli) [inline src/connections/destinations/catalog/actions-metronome/index.md:17]:
```suggestion
2. Search for "Metronome (Actions)" within the Destinations Catalog and select **Metronome (Actions)**
```

File: src/connections/destinations/catalog/actions-metronome/index.md [hunk 3]
```diff
@@ -11,13 +11,13 @@ redirect_from:
...
 3. Click **Configure Metronome (Actions)**.
-4. Select the source you’d like to connect to and give the destination a name
-5. Enter your Metronome API Token into the Segment Connection Settings UI (save changes).
+4. Select the source you’d like to connect to and give the destination a name (e.g., Metronome Instance #1).
```
Comment (markzegarelli) [inline src/connections/destinations/catalog/actions-metronome/index.md:19]:
```suggestion
4. Select the source you’d like to connect to and give the destination a name (for example, `Metronome Instance #1`).
```

**Review by markzegarelli — CHANGES_REQUESTED** [review]
Thanks @mcough2! A few small updates, and we'll roll this out with our next deploy on Tuesday,

File: src/connections/destinations/catalog/actions-metronome/index.md (hunk 2 above)
Comment (stayseesong) [reply src/connections/destinations/catalog/actions-metronome/index.md:17]: I think this should be kept as bolded rather than adding ""

**Pull request merged**